histomicstk/cli/NucleiDetection/NucleiDetection.py

```diff
@@ -383,8 +383,11 @@ def main(args):
         os.path.basename(args.outputNucleiAnnotationFile))[0]
 
     annotation = {
-        'name': annot_fname + '-nuclei-' + args.nuclei_annotation_format,
-        'elements': nuclei_list
+        'name': 'NucleiDetection',
+        'src_mu_lab': src_mu_lab,
```

Comment:
For the annotation to validate, we want these additional fields to be in the `attributes` key.  For consistency with PositivePixelCount, we should call the input parameters just `params`.  And, let's not change the output name as it has some useful information in it.  That is:
```
   annotation = {
        'name': annot_fname + '-nuclei-' + args.nuclei_annotation_format,
        'elements': nuclei_list,
        'attributes': {
            'src_mu_lab': src_mu_lab,
            'src_sigma_lab': src_sigma_lab,
            'params': vars(args),
        },
}
```